examples/librispeech/s0/conf/train_conformer.yaml

```diff
@@ -47,6 +47,8 @@ collate_conf:
     speed_perturb: false
     feature_extraction_conf:
         feature_type: 'fbank'
+        from_gpu: true
+        sample_rate: 16000
```

Comment:
add a resample bool flag?
Maybe we will mix the 8k and 16k training by resampling or just ignoring the sample rate in the future.

Reply:
and any results of the speedup by using GPU for fbank computation?